src/test/skript/tests/syntaxes/effects/EffOperations.sk

```diff
@@ -0,0 +1,142 @@
+test "operations effect numbers by literal numbers":
```

Comment:
still missing tests for null operands, tests where non-variable expressions are changed, etc

Reply:
unresolved